ctk/idp/src/main/kotlin/org/codice/compliance/web/sso/RedirectSSOTest.kt

```diff
@@ -178,21 +158,17 @@ class RedirectSSOTest : StringSpec() {
 
             // Get response from AuthnRequest
             val response = sendRedirectAuthnRequest(queryParams)
-            BindingVerifier.verifyHttpStatusCode(response.statusCode)
-
-            // Get response from plugin portion
-            val idpResponse = getServiceProvider(IdpSSOResponder::class)
-                    .getRedirectResponse(response).decorate()
-            // TODO When DDF is fixed to return a POST SSO response, uncomment this line
-            // SingleSignOnProfileVerifier.verifyBinding(idpResponse)
-            idpResponse.bindingVerifier().verify()
+            val finalHttpResponse =
+                    getServiceProvider(IdpSSOResponder::class).getResponseForRedirectRequest(
+                            response)
+//            SingleSignOnProfileVerifier.verifyBinding(finalHttpResponse)
```

Comment:
✏️ Should probably include a todo comment here indicating why this is commented out and when/how it will be enabled.